src/tests/ftest/get_remote_files.sh

```diff
@@ -0,0 +1,237 @@
+
+#######################################
+# Print usage to user.
+#######################################
+usage() {
+    echo -n "get_remote_files.sh [OPTION]...
+
+This script has 3 modes that can be executed independently if needed.
+
+ 1. Display big files specified by threshold value i.e. -t 5mb. This option
+    will list files within provided local path with -d option and will prepend
+    each file with a \"Y\" if file exceeds threshold value and with \"N\" if
+    it does not.
+
+ 2. Compress files specified in provided local path with -d option that exceed
+    1M in size. This script will check if current local system is a VM host, if
+    so, compression will not be executed.
+
+ 3. Archive files specified in provided local path with -d option to provided
+    remote path destination.
+
+ Options:
+      -a        Archive files
+      -c        Compress files
+      -d        Local path to files for achiving/compressing/size checking
+      -r        Remote path to scp files to i.e. server-A:/path/to/file
+      -t        Threshold value to determine classification of big logs
+      -v        Display commands being executed
+      -h        Display this help and exit
+"
+}
+
+#######################################
+# Print set GLOBAL variables.
+#######################################
+display_set_vars() {
+    echo "Set Variables:
+    ARCHIVE = ${ARCHIVE}
+    COMPRESS = ${COMPRESS}
+    REMOTE_DEST = ${REMOTE_DEST}
+    LOCAL_SRC = ${LOCAL_SRC}
+    THRESHOLD = ${THRESHOLD}
+    VERBOSE = ${VERBOSE}"
+}
+
+#######################################
+# Convert human readable size values to respective byte value.
+# Arguments:
+#   size: size value to convert to bytes. i.e. 5mb, 40gb, 6k, 10g
+# Returns:
+#   converted byte value.
+#######################################
+human_to_bytes() {
+    h_size=${1,,}
+
+    declare -A units=(
+        [b]=1
+        [kb]=$(python -c "print (2**10)")
+        [k]=$(python -c "print (2**10)")
+        [mb]=$(python -c "print (2**20)")
+        [m]=$(python -c "print (2**20)")
+        [gb]=$(python -c "print (2**30)")
+        [g]=$(python -c "print (2**30)")
+    )
+    if [[ "${h_size,,}" =~ ([0-9.]+)([kbmgt]+) ]]; then
+        val="${BASH_REMATCH[1]}"
+        unit="${BASH_REMATCH[2]}"
+        (( val = val*${units[${unit}]} ))
+        echo "${val}"
+    else
+        echo "Invalid value provided: ${1}"
+        return 1
+    fi
+}
+
+#######################################
+# List and tag files with a 'Y' or 'N' to classify
+# big files based on the threshold provided.
+# Arguments:
+#   logs_dir: path to local logs
+#   threshold: value to use as threhold to classify big files
+# Returns:
+#   stdout output listing the files prepended with a 'Y' or 'N' tag
+#######################################
+list_tag_files() {
+    # shellcheck disable=SC2016,SC1004
+    awk_cmd='{ \
+    if (th <= $1){ \
+        print "Y:",hostname,$1,$2} \
```

Comment:
I'd remove the hostname so we can get combined clush output:
```suggestion
        print "Y:",$1,$2} \
```

Reply:
Done.